### PR TITLE
Feature/gov page

### PR DESCRIPTION
**Pull request by rockacola**
Update heading of "NEO Government" page:

* Better align with design (v0.07) against various of viewports
* Implement a reusable partial `IntroductionSplash`, that can later be adapt into other pages for consistencies.

### Current design:

image

### PR design:

image

### CODE REVIEW

File: NeoWeb/Views/Gov/Index.cshtml [hunk 1]
```diff
@@ -11,15 +11,13 @@
...
-        <div class="col-lg-6">
-            <h1 class="uppercase mt-5"><span class="green">NEO</span> <br class="@SharedLocalizer["d-none-zh"]" /> @Localizer["Governance"]</h1>
-            <p>@Localizer["NEO token holders decide who is in charge of maintaining the Neo network through the election of a Neo Council. GAS token rewards are distributed to voters and council members alike."]</p>
-        </div>
-        <div class="col-lg-4 offset-lg-1">
-            <img class="img-fluid" src="@Helper.CDN/images/gov/gov.svg" />
-        </div>
-    </div>
+    @{
+        ViewData["IS_title"] = @Localizer["<em>NEO</em><br/>Governance"]; // TODO: i18n consideration
```
Comment (chenzhitong) [inline NeoWeb/Views/Gov/Index.cshtml]: Need to update resources file.

File: NeoWeb/Views/Gov/Index.cshtml [hunk 2]
```diff
@@ -11,15 +11,13 @@
...
+        ViewData["IS_description"] = @Localizer["NEO token holders decide who is in charge of maintaining the Neo network through the election of a Neo Council. GAS token rewards are distributed to voters and council members alike."];
+        ViewData["IS_imagePath"] = @Helper.ToCDN("/images/gov/gov.svg", true);
+    }
+    <partial name="../_Common/_IntroductionSplash" view-data="ViewData" />
```
Comment (chenzhitong) [inline NeoWeb/Views/Gov/Index.cshtml:19]: No need to create partial views here

File: NeoWeb/wwwroot/css/_introduction-splash.less [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      flex-shrink: 0;
+      flex-basis: 100%;
+
+      @media (min-width: 1024px) {
+          margin-top: 36px;
+          flex-basis: 60%;
+      }
+
+      .introduction-title {
+          font-size: @font-size-42px;
```
Comment (chenzhitong) [inline NeoWeb/wwwroot/css/_introduction-splash.less:24]: Missing variable declaration

File: NeoWeb/Views/Gov/Index.cshtml (hunk 1 above)
Comment (chenzhitong) [reply NeoWeb/Views/Gov/Index.cshtml]: In this line, it is better not to change the line in the Chinese case.

**Review by chenzhitong — CHANGES_REQUESTED** [review]
Update resources file;
Chinese style of title;
Remove partial view.

**Comment by rockacola** [comment]
I can look into the other parts, but with:

> Remove partial view.

I'll prefer to have the partial set up so that the other pages (such as [Developer](https://neo.org/dev), [Eco Support](https://neo.org/eco)) can reuse the same UI component.

**Pull request merged**